### ISSUE TITLE
I want to add the source code button instead of "Click here for the code" text. 

### ISSUE THREAD

**Issue by Babunshil** [Arman-ali-khan-786/CardUI#129]
https://user-images.githubusercontent.com/106283743/231211083-baa1fde1-c4c5-432d-8ec6-29f67c062cf6.mp4

**Comment by Babunshil** [comment]
kindly suggest me the button colour as you want .

**Comment by Arman-ali-khan-786** [comment]
> kindly suggest me the button colour as you want .

yes you can do that and regarding the color light pink or header color (light) will be okk

**Comment by Babunshil** [comment]
Thank you
